src/Mod/Draft/draftviewproviders/view_hatch.py

```diff
@@ -26,7 +26,9 @@
 import PySide.QtCore as QtCore
 import PySide.QtGui as QtGui
 
-import FreeCADGui as Gui
+import FreeCAD as App
+if App.GuiUp:
+    import FreeCADGui as Gui
```

Comment:
view providers are normally never created when FreeCAD runs without the GUI. Are you sure this file is used? I think if yes, it might be good to find where it is called and fix that instead...

Reply:
Hi @yorikvanhavre you are right, view provider was initiated in `src/Mod/Draft/draftmake/make_hatch.py`. Here is the new commit 9f8f7c5ab3118264d418f1f94f06bd8ec61f1a63